src/main/java/Duke/Duke.java

```diff
@@ -0,0 +1,123 @@
+package Duke;
+
+import java.util.Scanner;
+
+public class Duke {
+    private static Task[] tasks = new Task[100];
+    private static int numberOfTasks;
+
+    private static String logo = "     ____        _        \n" +
+            "    |  _ \\ _   _| | _____ \n" +
+            "    | | | | | | | |/ / _ \\\n" +
+            "    | |_| | |_| |   <  __/\n" +
+            "    |____/ \\__,_|_|\\_\\___|\n";
+
+    private static String SEPARATION_LINE = "    ____________________________________________________________";
+
+    private static String SPACES = "       ";
```

Comment:
Good job. Clear naming of constant variables.

Reply:
Thank you for your compliment! :)